### DISCUSSION TITLE
Choosing the render camera

### DISCUSSION THREAD

**Discussion by matiascodesal** [GPUOpen-LibrariesAndSDKs/BlenderUSDHydraAddon#25] [Ideas]
I think the UX for choosing the camera to render with could be improved.  Right now it seems like it uses Blender's active camera, but that only works if you're including blender data.  How would you specify a camera coming from a USD layer?

I think I got lucky and named my USD camera the same as my blender camera so it proceeded without error rendering with the USD camera.

Maybe the camera should be specified in the render engine so you can present the user with options based on blender and USD data?

**Comment by bnagirniak** [comment]
Thank you for the idea, we'll take into account. Can't answer exactly to your questions right now.

**Comment by bsavery** [comment]
Again, should this be done Solaris style in a node?  That's my general thought.

**Reply by matiascodesal** [reply]
I haven't seen how Solaris does it yet, but that would make sense.  I think it could even go on the "Render USD via Hydra" node.  Either as a primpath taken as input or a dropdown on that node prefilled with  cameras found in the stage.

So, if you're using a NodeTree, you specify the camera via nodes.  If you choose, "Scene" as the data source, you use the active camera.
